classes/Exception/ThankYouRepository.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace Claromentis\ThankYou\Exception;
+
+class ThankYouRepository extends ThankYouException
```

Comment:
This is unused and can be removed. If it was in use it should definitely be renamed, as it shares the name of the actual `ThankYouRepository`, which can be a pain for searching. 😄

Reply:
Fixed in 6287718fd32b2d0c5b26ff98534653bee5bad620